src/sentry/integrations/message_builder.py

```diff
@@ -192,10 +192,7 @@ def build_footer(
         text = rules[0].label if rules[0].label else "Test Alert"
         footer += f" via {url_format.format(text=text, url=rule_url)}"
 
-        if (
-            features.has("organizations:slack-block-kit", project.organization)
-            and url_format == SLACK_URL_FORMAT
-        ):
+        if url_format == SLACK_URL_FORMAT:
```

Comment:
need to do this because only slack uses this footer style

Reply:
could we just override this method in the slack class instead? Or does it have to go in this specific order/location?

Reply:
hmm... i guess we could https://github.com/getsentry/sentry/blob/7ee63b5c3c4db7e0464c929b734a2c250e9b1117/src/sentry/integrations/slack/message_builder/issues.py#L526

my thinking of putting the logic here is that it isolates all footer logic to one function

Reply:
*footer logic other than specific notifications, i noticed there's a `build_notification_footer` too 😵‍💫